### PR TITLE
Fix front end tests

### PR DESCRIPTION
**Pull request by bwreid**
Covers a number of cards. Primarily [this card](https://www.pivotaltracker.com/story/show/97112720) which is about fixing the front-end tests and is mostly solved with the first three commits. The following commits solve missing pages in order to eventually fix failing tests:

[Pivotal Card I](https://www.pivotaltracker.com/story/show/101219878)
[Pivotal Card 2](https://www.pivotaltracker.com/story/show/101215438)
[Pivotal Card 3](https://www.pivotaltracker.com/story/show/101214658)
[Pivotal Card 4](https://www.pivotaltracker.com/story/show/101211676)

### CODE REVIEW

File: pg-setup/database-setup.sql [hunk 1]
```diff
@@ -0,0 +1,1790 @@
...
+--
+
+CREATE TABLE ballot_candidates (
+    results_id bigint NOT NULL,
+    ballot_id bigint,
+    candidate_id bigint
+);
+
+
+ALTER TABLE ballot_candidates OWNER TO wesreid;
```
Comment (tie-rack) [inline pg-setup/database-setup.sql]: `wesreid`?

Comment (bwreid) [inline pg-setup/database-setup.sql]: Yeah, that's not right. I can remove those comment lines but what to do about `OWNER TO`? It looks as though they're not needed to correctly run the import so maybe just remove?

Comment (tie-rack) [inline pg-setup/database-setup.sql]: Yeah. If it works without `OWNER TO`, getting rid of them is probably cool.

**Comment by tie-rack** [comment]
Great. LGTM. This will come in handy!

**Pull request merged**